trellis-app-triplestore/build.gradle

```diff
@@ -8,48 +8,50 @@ mainClassName = 'org.trellisldp.app.triplestore.TrellisApplication'
-    compile project(':trellis-agent')
-    compile project(':trellis-audit')
-    compile project(':trellis-event-serialization')
-    compile project(':trellis-webac')
-    compile project(':trellis-triplestore')
-    compile project(':trellis-jms')
-    compile project(':trellis-kafka')
-    compile project(':trellis-app')
-
-    runtime group: 'javax.xml.bind', name: 'jaxb-api', version: jaxbVersion
-
-    testCompile group: 'io.dropwizard', name: 'dropwizard-client', version: dropwizardVersion
-    testCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: dropwizardVersion
-    testCompile group: 'org.apache.activemq', name: 'activemq-broker', version: activeMqVersion
-    testCompile group: 'org.apiguardian', name: 'apiguardian-api', version: apiguardianVersion
-    testCompile group: 'org.apache.tamaya', name: 'tamaya-core', version: tamayaVersion
-    testCompile group: 'org.awaitility', name: 'awaitility', version: awaitilityVersion
-    testCompile group: 'ch.qos.logback', name: 'logback-classic', version: logbackVersion
-    testCompile group: 'org.mockito', name: 'mockito-core', version: mockitoVersion
-    testCompile project(':trellis-test')
+    implementation("com.github.jsonld-java:jsonld-java:$jsonldVersion") {force = true}
+    implementation("commons-codec:commons-codec:$commonsCodecVersion")
+    implementation("io.dropwizard:dropwizard-auth:$dropwizardVersion")
+    implementation("io.dropwizard:dropwizard-core:$dropwizardVersion")
+    implementation("io.dropwizard:dropwizard-metrics:$dropwizardVersion")
+    implementation("io.dropwizard:dropwizard-http2:$dropwizardVersion")
+    implementation("io.jsonwebtoken:jjwt:$jwtVersion")
+    implementation("javax.activation:javax.activation-api:$activationApiVersion")
+    implementation("javax.jms:javax.jms-api:$jmsApiVersion")
+    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    //TODO resolve activemq and jena-osgi for JPMS
+    implementation("org.apache.activemq:activemq-client:$activeMqVersion")
+    implementation("org.apache.jena:jena-osgi:$jenaVersion")
+    implementation("org.apache.jena:jena-tdb2:$jenaVersion")
+    implementation("org.slf4j:slf4j-api:$slf4jVersion")
+
+    implementation project(':trellis-constraint-rules')
+    implementation project(':trellis-io-jena')
+    implementation project(':trellis-id')
+    implementation project(':trellis-api')
+    implementation project(':trellis-http')
+    implementation project(':trellis-vocabulary')
+    implementation project(':trellis-file')
+    implementation project(':trellis-namespaces')
+    implementation project(':trellis-agent')
+    implementation project(':trellis-audit')
+    implementation project(':trellis-event-serialization')
+    implementation project(':trellis-webac')
+    implementation project(':trellis-triplestore')
+    implementation project(':trellis-jms')
+    implementation project(':trellis-kafka')
+    implementation project(':trellis-app')
```

Comment:
Because this is a deployable application (rather than a library), I'd prefer to use the `compile` scope for the above dependencies.

Reply:
I think that compile and testCompile are deprecated.  the only current options are api and implementation.  Check the docs here: https://docs.gradle.org/current/userguide/java_library_plugin.html

Reply:
That’s for the java-library plugin. This should be using the java plugin

Reply:
At present, the root build script applies the `java` and `java-library` plugin to _all_ subprojects. It would probably be better to selectively apply those plugins to the individual projects, as is currently done with the `osgi` plugin.

Reply:
I removed the java plugin from the root build script and applied it only to `trellis-app-triplestore` and `trellis-osgi`.  These are the only modules that require `compile`, correct?  I understand your motivation here.